fastserde/avro-fastserde/build.gradle

```diff
@@ -49,7 +49,7 @@ dependencies {
   implementation "org.apache.commons:commons-lang3:3.4"
   implementation "com.sun.codemodel:codemodel:2.6"
 
-  compile ("org.apache.avro:avro:1.10.2") {
+  compileOnly ("org.apache.avro:avro:1.10.2") {
```

Comment:
I wonder if we can use "java11CompileOnly" for jdk11 classes Avro dependency and remove custom pom logic. But need to test whether declared dependency on Avro will re-appear in POM file. If not - this will reduce amount of build customizations

Reply:
yes, compileOnly should avoid exporting the dependency, which means it shouldnt be in the pom.xml

Reply:
Could you please run publishToMavenLocal and verify that resulting pom file does not have dependency on Avro? Sorry, just want to make sure we do not resurface this problem. Thanks

Reply:
i have, and there is no avro dependency in the pom file for fast-serde

Reply:
Thanks